include/swift/Basic/LangOptions.h

```diff
@@ -104,6 +104,10 @@ namespace swift {
     /// Enable features useful for running in the debugger.
     bool DebuggerSupport = false;
 
+    /// Enable the MemortBufferSerializerdModileImporter.
+    /// Only used by lldb-moduleimport-test.
```

Comment:
Typo. Also, it's used by the real LLDB too!

Reply:
Technically, LLDB does not actually set this flag, it just creates and installs a memory buffer loader on its own.

Reply:
Ahh, fair point.